frontend/packages/helm-plugin/src/components/forms/install-upgrade/HelmInstallUpgradePage.tsx

```diff
@@ -151,10 +150,8 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
       // eslint-disable-next-line global-require, @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
       ajv.addMetaSchema(require('ajv/lib/refs/json-schema-draft-06.json'));
```

Comment:
If we don't call ajv.validateSchema anymore, we don't need the ajv configuration and import in this file!

Reply:
Updated the PR. PTAL.

For error message I updated text as "Errors in the form data." Is this fine?